src/Components/Components/test/NavigationManagerTest.cs

```diff
@@ -659,101 +660,63 @@ async ValueTask HandleLocationChanging(LocationChangingContext context)
     }
 
     [Fact]
-    public async Task LocationChangingHandlers_CanBubbleUpExceptionsSynchronously_WhenOneHandlerIsRegistered()
```

Comment:
Tests verifying that exceptions bubble up through `NotifyLocationChangingAsync()` are no longer relevant because they're now handled in `HandleLocationChangingHandlerException()` instead. If we kept this original behavior, we would risk surfacing the exception in more than once place and logging more than once for the same exception.

To replace these tests, I've added some others that verify that `HandleLocationChangingHandlerException()` is properly invoked and exceptions are _not_ bubbled through `NotifyLocationChangingAsync()`.